FlyingFox/Tests/Handlers/HTTPHandlerTests.swift

```diff
@@ -135,10 +135,10 @@ struct HTTPHandlerTests {
             FileHTTPHandler.makeContentType(for: "fish.css") == "text/css"
         )
         #expect(
-            FileHTTPHandler.makeContentType(for: "fish.js") == "application/javascript"
+            FileHTTPHandler.makeContentType(for: "fish.js") == "text/javascript"
```

Comment:
We also need to update the XCTests which only run when building with Swift 5

https://github.com/swhitty/FlyingFox/blob/main/FlyingFox/XCTests/Handlers/HTTPHandlerTests.swift